client-common/src/main/java/org/apache/cassandra/sidecar/common/utils/StringUtils.java

```diff
@@ -46,4 +48,12 @@ public static boolean isNotEmpty(@Nullable String string)
     {
         return !isNullOrEmpty(string);
     }
+
+    public static boolean contains(@NotNull String string, @NotNull String target) {
```

Comment:
I do not see a reason why we should have the StringUtils class. 

For all other methods in this class can be removed and Guava StringUtils could be used, for this method can use `String.contains()` since the strings are not null.

If you think that the strings could be null, then there is a mismatch in the annotation for parameters

Reply:
Short answer to "why we should have the StringUtils class" is that we do not want to add the `guava` dependency in this package. Analytics depends on `client-common`

Reply:
`guava` could cause some issues when pulling it into spark for example. `client-common` needs to be lean and have the minimum number of dependencies.